dali/operators/audio/mel_scale/mel_filter_bank.cc

```diff
@@ -24,7 +24,7 @@ DALI_SCHEMA(MelFilterBank)
 triangular filters.
 
 The frequency ('f') dimension is selected from the input layout.
-In case of no layout, "f", "ft", or "*ft" is assumed, depending on the number of dimensions.
+In case of no layout, "f", "ft", or "\*ft" is assumed, depending on the number of dimensions.
```

Comment:
If we can afford to run the CI once again, there's a typo a little bit below:

If this value is not provided, ``sample_rate /2`` is used.

to

If this value is not provided, ``sample_rate/2`` is used.

Reply:
Done